In [13]:
import numpy as np
CONV = 4.135668 # h * 1 GHz = 4.135668 µeV
pinchoff = (np.sqrt(2 * 75.5 * 29.1))
zeroflux = (np.sqrt(2 * (75.5 + 150 * 0.55 / 4) * 29.1))
piflux = (np.sqrt(2 * (75.5 - 150 * 0.55 / 4) * 29.1)) 

In [18]:
(zeroflux - piflux)/CONV

4.4208508944154197

Let me measure all energies in $\mu$eV. From the table in the supplementary I have $E_J = 75.5\,\mu$eV, $E_C=29.1\,\mu$eV. This means that at pinch-off:

\begin{equation}
h\omega_\textrm{pinchoff}\,=\,\sqrt{2 E_C E_J} \approx 66.28 \,\mu\textrm{eV}
\end{equation}

When I convert to GHz, I indeed obtain $\omega_\textrm{pinchoff} = 16.02$ GHz as quoted in the Supplementary.

Now let's compute approximately the plasma frequency away from pinch-off, for a gate voltage close to that in panel a of Fig. 4. For that one we estimate $\Delta\approx 150\,\mu$eV and $T\approx 0.55$. The plasma frequency is now a function of flux $\varphi$. In particular, by expanding the Josephson potential we may estimate its maximum value at $\varphi=0$ and its minimum value at flux $\varphi=\pi$.

We get

\begin{align}
h\omega_{\varphi=0}\,=\,\sqrt{2 E_C (E_J + \Delta T /4)} \approx 74.79 \,\mu\textrm{eV}\\
h\omega_{\varphi=\pi}\,=\,\sqrt{2 E_C (E_J - \Delta T /4)} \approx 56.51 \,\mu\textrm{eV}
\end{align}

Note: this actually assumes that at $\varphi=\pi$ the Josephson potential has a single minimum. This may actually not be the case given the energy scales of the circuit (recall the double well in Gijs' paper). However, for the moment let's neglect this complication.

You see that as a function of the plasma frequency the range of oscillation of $\omega_p$ is given by:

\begin{equation}
h\delta\omega\,=h(\omega_{\varphi=0}-\omega_{\varphi=\pi}) \approx 18.28 \,\mu\textrm{eV}
\end{equation}

In GHz, the variation is given by $\delta\omega \approx 4.42$ GHz.

With comparison to the data, we must also include a factor of 2 due to the relation $2eV=\omega$ for the ICPT. This means that the variation of the plasma frequency with flux should be visibile as an oscillation of about

$$\frac{h\delta\omega}{2e}\approx 9.14\,\mu\textrm{V}.$$

This is roughly what I see in the data scan: the LC line oscillates by about $10\,\mu$V. So that seems to be OK.

What does not agree is the value of the plasma frequency at zero flux. To extract this value directly from the data, I look at the position of maximum intensity of the signal for $\varphi\approx 0$ in the $B=0$ data scan that you sent me. This way, I would estimate:

\begin{align}
h\omega^\textrm{measured}_{\varphi=0}\,\approx 58.46 \,\mu\textrm{eV}\\
\end{align}

So as you can see, the problem is that with $E_J\approx 75.5\,\mu$eV and $E_C\approx 29.1\,\mu$eV, we obtain the problematic conclusion:

\begin{align}
\omega_\textrm{pinchoff} > \omega^\textrm{measured}_{\varphi=0}
\end{align}

I don't understand how this can be the case. One would of course expect the plasma frequency at zero flux to be larger than that at pinch-off. Not surprisingly, my fits are off because they overestimate $\omega_{\varphi=0}$. 

Possible resolutions (not to rule out anything):

* Either $E_J$ or $E_C$ are overestimated.
* The voltage vlaues of the data set I am using is off.
* My extraction of the plasma frequency value is wrong. However, I simply find a maximum and read out the corresponding voltage value, so I don't see how it can be wrong by more than a few percent. In the plots for Fig. 3, this simple procedure seems to work fine.

I think I recall that you guys have a way to extract the curve $\omega(\varphi)$ by fitting the LC peak. Which value of $\omega_{\varphi=0}$ do you extract? Would it lead to a different conclusion?